subsys/bluetooth/controller/ll_sw/ull_df.c

```diff
@@ -802,17 +800,17 @@ static bool pdu_ext_adv_is_empty_without_cte(const struct pdu_adv *pdu)
  * advertising chain. If particular PDU is empty (holds cte_info only) it will be removed from
  * chain.
  *
- * @param lll_sync Pointer to periodic advertising sync object.
- * @param pdu_prev Pointer to a PDU that is already in use by LLL or was updated with new PDU
- *                 payload.
- * @param pdu      Pointer to a new head of periodic advertising chain. The pointer may have
- *                 the same value as @p pdu_prev, if payload of PDU pointerd by @p pdu_prev was
- *                 already updated.
+ * @param lll_sync      Pointer to periodic advertising sync object.
```

Comment:
Not sure this is needed, but at least often we use both [in] and [out], or none ([in] is implied when nothing is added, although the same can be said here). 
```suggestion
 * @param[in] lll_sync      Pointer to periodic advertising sync object.
```

Reply:
applied suggestion